mitodl/mitxonline
# Refine app

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #519 

#### What's this PR do?
This adds a barebones refine application for local development only for now.

#### How should this be manually tested?
Run `docker-compose up` and verify you can reach the dashboard at http://mitxonline.odl.local:8016/
